src/System.Text.Json/src/System/Text/Json/Serialization/ReadStack.cs

```diff
@@ -7,6 +7,7 @@
 
 namespace System.Text.Json.Serialization
 {
+    [DebuggerDisplay("Current: ClassType.{Current.JsonClassInfo.ClassType} {Current.JsonClassInfo.Type.Name}")]
```

Comment:
nit: clean up debugger helpers.

Reply:
Clean up how? I don't intend to remove them.